booking/src/main/java/org/dcsa/conformance/standards/booking/BookingScenarioListBuilder.java

```diff
@@ -78,101 +78,131 @@ private BookingScenarioListBuilder thenAllPathsFrom(BookingState bookingState) {
   private BookingScenarioListBuilder thenAllPathsFrom(
       BookingState bookingState, BookingState originalBookingState) {
     return switch (bookingState) {
-      case CANCELLED, COMPLETED, DECLINED, REJECTED  -> then(shipper_GetBooking(bookingState));
+      case COMPLETED  -> then(
+        shipper_GetBooking(bookingState).thenEither(
+         uc7_shipper_submitBookingAmendment(true)
```

Comment:
General remark: Please rename and use a different name for abuse test cases when they are used in the state transitions.

In the eBL, I named them `auc_X` (for `AbUse Case`).

this makes it easier to see at a distance that these transitions are invalid.


Note: I am fine with them being funneled into the same underling code (action classes) after that. The point is that I want to quickly be able to tell "valid" from "invalid" transition apart something as complex as this. A `true` / `false` will not cut it as `true` / `false` are non-descriptive (especially when the editor is not there to provide inlay hints, which it is not in GitHub among other)